ztest/ztest.go

```diff
@@ -327,6 +327,11 @@ func (z *ZTest) check() error {
 	} else {
 		return errors.New("either a zql field or script field must be present")
 	}
+	if z.ErrorRE != "" {
+		var err error
+		z.errRegex, err = regexp.Compile(z.ErrorRE)
```

Comment:
This renders the `regexp.Compile` call in `FromYAMLFile` redundant, so maybe remove that?